README.md

```diff
@@ -38,36 +39,39 @@ When you have all that installed you can simply do the following:
 ```
 git clone --recurse-submodules https://github.com/DSD-DBS/capella-collab-manager.git
 cd capella-collab-manager
-make create-cluster  # this creates cluster and container registry
 ```
 
-At this point, have a look at the [frondend README](frontend/README.md) in order to configure the frontend.
-You'll need an `environment.prod.ts` file in the right location.
+Create a local K3D cluster and deploy the application
 
-To build the Capella images you'll need to download Capella (binary!) and copy the archive to `capella-dockerimages/capella/archives`. For instructions, please refer to the [capella-dockerimages' README](capella-dockerimages/README.md).
+```
+make create-cluster
+MY_EMAIL=me@example.com make deploy
+```
 
-Once you've configured the frontend and capella images, continue with your deployment:
+Replace `me@example.com` with your e-mail address. An administrator account will be made for this user.
```

Comment:
```suggestion
Replace `username` with a username. An administrator account will be made for this user. Make sure you log in with the username if you want to test the admin account.
```

According to the last comment, we should use `username` instead of the email here. Also, I added a note to make clean that you can use every username when using the oauth mock.